lib/web_ui/lib/src/engine/compositor/canvaskit_api.dart

```diff
@@ -1705,14 +1705,14 @@ class SkImageInfo {
     SkColorSpace colorSpace,
     SkColorType colorType,
   });
-  external SkAlphaType get alphaType();
-  external SkColorSpace get colorSpace();
-  external SkColorType get colorType();
-  external int get height();
-  external bool get isEmpty();
-  external bool get isOpaque();
-  external SkRect get bounds();
-  external int get width();
+  external SkAlphaType alphaType();
+  external SkColorSpace colorSpace();
+  external SkColorType colorType();
+  external int height();
+  external bool isEmpty();
+  external bool isOpaque();
+  external SkRect bounds();
+  external int width();
```

Comment:
These should be getters I imagine. However, getters don't need the `()`, for example `external bool get isEmpty;` (notice no `()` at the end).

Reply:
I don't know any more. The doc I finally found explicitely says they are functions, using ().

Reply:
Hmm. Interesting. According to the `@JS` bindings it's a plain object (in JS you'd write `{ width: x, height: y, aphaType: z, colorSpace: a, colorType: b }`). Perhaps there's a difference between `SkImageInfo` constructed by JS and one returned by CanvasKit?

Why don't you do what works now, and we can clean this up later?

/cc @kjlubick

Reply:
According to CanvasKit bindings it should be a POJO: https://github.com/google/skia/blob/1d4b92d87806e0d3ed8b4155da98d7d0ac757bd6/modules/canvaskit/canvaskit_bindings.cpp#L1873

That is, there shouldn't be any methods/functions on `SkImageInfo`, only properties.

Reply:
That is correct, the SkImageInfo is just a POJO

Reply:
OK, I do, I do.